sdk/storage/azdatalake/sas/service.go

```diff
@@ -406,7 +450,7 @@ func parsePathPermissions(s string) (PathPermissions, error) {
 		case 'p':
 			p.Permissions = true
 		default:
-			return PathPermissions{}, fmt.Errorf("invalid permission: '%v'", r)
+			return FilePermissions{}, fmt.Errorf("invalid permission: '%v'", r)
```

Comment:
this is an internal method that always converts the permissions to a string to just default to FilePermissions